packages/lib/server/repository/credits.ts

```diff
@@ -95,26 +95,45 @@ export class CreditsRepository {
     }
   }
 
-  static async findCreditBalanceWithExpenseLogs({ teamId }: { teamId: number }, tx?: PrismaTransaction) {
+  static async findCreditBalanceWithExpenseLogs(
+    {
+      teamId,
+      userId,
+      startDate = dayjs().startOf("month").toDate(),
+      endDate = new Date(),
+      creditType,
+    }: { teamId?: number; userId?: number; startDate?: Date; endDate?: Date; creditType?: CreditType },
+    tx?: PrismaTransaction
+  ) {
+    if (!teamId && !userId) return null;
+
     const prismaClient = tx ?? prisma;
```

Comment:
Can you remind me why we do this? Sorry I'm not familiar with `PrismaTransaction`